README.md

```diff
@@ -573,6 +573,79 @@ Don't worry about `adbkey` or `adbkey.pub` under `/.android`, not required.
 
   > Unfortunately it is not possible to pass through a USB device (or a serial port) to a container.
 
+## Firebase Test Lab 
+You can also run your UI tests on [Firebase Test Lab](https://firebase.google.com/docs/test-lab) using emulators or physical devices, to do so first you need to
+
+* Create a project in [Google Cloud Platform](https://console.cloud.google.com/cloud-resource-manager) if you haven't created one yet.
+* Create a project in [Firebase](https://console.firebase.google.com/):
+    * Choose the recently created Google Cloud Platform project to add Firebase services to it.
+    * Confirm Firebase billing plan.
+* Go to **IAM & Admin** -> **Service Accounts** in [Google Cloud Platform](https://console.cloud.google.com/iam-admin/serviceaccounts):
+    * Edit the Firebase Admin SDK Service Agent account.
+    * **Keys** -> **ADD KEY** -> **Create new key** -> **Key type**: **JSON** -> **CREATE**.
+    * Download and save the created private key to your computer.
+* Go to **IAM & Admin** -> **IAM** in [Google Cloud Platform](https://console.cloud.google.com/iam-admin/iam):
+    * Edit the Firebase Admin SDK Service Agent account.
+    * **ADD ANOTHER ROLE** -> **Role**: **Project** -> **Editor** -> **SAVE**.
+* Go to [API Library](https://console.developers.google.com/apis/library) -> search for **Cloud Testing API** and **Cloud Tool Results API** -> enable them.
+
+* Now let's pull and run the Docker image with Google Cloud SDK
```

Comment:
The first part is talking about GCP & Firebase configuration, and the second part is about up and running in container, thus this should not be the same level as the other bullet points.  Please remove the `*` and probably rephrase it to:

`Now we're able to connect to Firebase Test Lab inside the container and deploy UI tests over there.`